### PR TITLE
Chart

### LINKED ISSUE #36: Chart

### PR DESCRIPTION
**Pull request by Cal9233**
close #36 
some pages organized,
empty chart added

### CODE REVIEW

File: client/src/App.jsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React, { Component } from 'react';
```
Comment (ErnieAtLYD) [inline client/src/App.jsx:1]: I'm pretty sure you don't need the `{ Component }` part unless you are explicitly calling this somewhere else in the file.

File: client/src/api.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import axios from 'axios';
+
+const fetchTask = async () => {
+    const url = 'http://localhost:3000/api/task';
```
Comment (ErnieAtLYD) [inline client/src/api.js:4]: Hey, so be super careful on this one. One slip up and all the sudden this is being pushed to Heroku with a call to localhost:3000 - better to probably save this without the hostname; I think create-react-app has a particular way of handling proxies as well that you may want to look into

File: client/src/api.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+    try {
+        const {
+            data: { Monday, Tuesday, Wednesday, Thursday, Friday, Saturday, Sunday },
```
Comment (ErnieAtLYD) [inline client/src/api.js:7]: Do you mean this to be strings or actual variables?

File: client/src/pages/Stats.jsx [hunk 4]
```diff
@@ -1,11 +1,67 @@
...
+
+    const graph = () => {
+      setLineChart({
+        labels: [ 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday', 'Sunday' ],
+        datasets: [
+          {
+            label: "Your Progress!",
+            data: 'insert session hours',
+            backgroundColor: [
+              'rgba( 75, 192, 192, 0.6'
```
Comment (ErnieAtLYD) [inline client/src/pages/Stats.jsx:17]: Just like this? No `)` at the end?

File: client/src/api.js (hunk 2 above)
Comment (Cal9233) [reply client/src/api.js]: ah yes, mistake on my part. I deleted the host name in a different branch but not in this one. My mistake

File: client/src/api.js (hunk 3 above)
Comment (Cal9233) [reply client/src/api.js]: Variables, I commented out the page. It's incomplete but it's a start up.

File: client/src/pages/Stats.jsx (hunk 4 above)
Comment (Cal9233) [reply client/src/pages/Stats.jsx]: oops

File: client/src/api.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+//             data: { Monday, Tuesday, Wednesday, Thursday, Friday, Saturday, Sunday },
+//         } = await axios.get(`${url}/api/tasks`);
+//         return { Monday, Tuesday, Wednesday, Thursday, Friday, Saturday, Sunday  }
+        
+//     } catch (error) {
+//         console.log(error)
+//     }
+// }
+
+// export default fetchTask;
```
Comment (leopolicastro) [inline client/src/api.js:16]: If we're not using this file let's delete it.

File: client/src/api.js (hunk 3 above)
Comment (ErnieAtLYD) [reply client/src/api.js]: @Cal9233 got it. Just keep in mind the rule about best practices in variable naming. If Monday ... Sunday are gonna be components or Classes, keep them uppercase. If not, use lowercase.

File: client/src/api.js (hunk 5 above)
Comment (Cal9233) [reply client/src/api.js]: branch pushed, fixed :)

**Pull request merged**